camp/camp-brooklyn/src/main/java/org/apache/brooklyn/camp/brooklyn/spi/creation/BrooklynEntityDecorationResolver.java

```diff
@@ -230,4 +237,51 @@ protected void addDecorationFromJsonMap(Map<?, ?> decorationJson, List<SpecParam
             throw new IllegalStateException("Not called");
```

Comment:
As per @tbouron's comment about whether to implement this - throwing an exception saying "Not called" is very misleading. If it is ever called, then telling the user that it wasn't called in the exception message will look weird!

Better to have a comment saying that it won't be called, and to throw an `UnsupportedOperationException` I think.

But it looks like this code already existed, rather than being modified in this PR (so doesn't need to be fixed here).